src/api/app/controllers/webui/requests_listing_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Webui::RequestsListingController < Webui::WebuiController
-  before_action :lockout_spiders, :require_login
+  before_action :lockout_spiders
```

Comment:
spiders don't log in...

Reply:
That's why you put `before_action :lockout_spiders` **_before_** `before_action :require_login`, right?

Reply:
`lockout_spiders` returns no content and `head :ok` based on the user-agent. This will be punished by search engines. A "please log in" page will not be punished by search engines.